messages/eslintrc-plugins.js

```diff
@@ -5,9 +5,14 @@ module.exports = function({ plugins }) {
     const isArrayOfStrings = typeof plugins[0] === "string";
 
     return `
-A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}.
+A config object has a "plugins" key defined as an array${isArrayOfStrings ? " of strings" : ""}. It
+looks something like this:
 
-Flat config requires "plugins" to be an object in this form:
+    {
+        "plugins": ${JSON.stringify(plugins)}
+    }
+
+Flat config requires "plugins" to be an object in this form, like this:
```

Comment:
```suggestion
Flat config requires "plugins" to be an object, like this:
```
last part seems repetitive, how about this one?